suite-native/config/src/supportedNetworks.ts

```diff
@@ -1,48 +1,42 @@
-import { networks, NetworkSymbol, getMainnets, getTestnets } from '@suite-common/wallet-config';
-
-const deprecatedNetworks = ['dash', 'btg', 'dgb', 'nmc', 'vtc'];
-
-// ordering defined by product - it is done according to the data about the coins and usage of them in Desktop Suite
-export const mainnetsOrder: NetworkSymbol[] = [
-    'btc',
-    'eth',
-    'ltc',
-    'doge',
-    'etc',
-    'ada',
-    'bch',
-    'xrp',
-    'dash',
-    'zec',
-    'btg',
-    'vtc',
-    'nmc',
-    'dgb',
-];
+import { A } from '@mobily/ts-belt';
+import { Network, NetworkSymbol, getMainnets, getTestnets } from '@suite-common/wallet-config';
+
+const discoveryBlacklist: NetworkSymbol[] = ['sol', 'dsol'];
+
+export const supportedPortfolioTrackerNetworks = {
+    mainnet: ['btc', 'eth', 'ltc', 'doge', 'etc', 'ada', 'bch', 'xrp', 'zec'] as NetworkSymbol[],
+    testnet: ['test', 'regtest', 'tsep', 'tgor', 'thol', 'tada', 'txrp'] as NetworkSymbol[],
+};
```

Comment:
This should be slighty safer because now you can try to access `supportedPortfolioTrackerNetworks.abc` and it won't throw error
```
export const supportedPortfolioTrackerNetworks = {
    mainnet: ['btc', 'eth', 'ltc', 'doge', 'etc', 'ada', 'bch', 'xrp', 'zec'] ,
    testnet: ['test', 'regtest', 'tsep', 'tgor', 'thol', 'tada', 'txrp'],
} as const satisfies Record<string NetworkSymbol[]>;
```

Reply:
70fe47a161